dialects/datafusion.yaml

```diff
@@ -26,19 +26,18 @@ dependencies:
   string: 
     https://github.com/substrait-io/substrait/blob/main/extensions/substrait/extensions/functions_string.yaml
 supported_types:
-  - i8
-  - i16
-  - i32
-  - i64
-  - fp32
-  - fp64
-  - bool
-  - str
-  - date
-  - time
-  - ts
-  - tstz
-  - interval
+  i8: int8
+  i16: int16
+  i32: int32
+  i64: int64
+  fp32: float32
+  fp64: float64
+  bool: bool_
+  str: string
+  date: timestamp
+  time: timestamp
+  ts: timestamp
+  tstz: timestamp
```

Comment:
How can these all map to the same type?

Reply:
I'm okay with these all mapping to timestamp for now since that's what the runner does.  I think the other date/time types weren't working properly with the date/time functions, so I added some handling based on timestamp length since timestamp could represent the others.  We can open up an issue to fix this and do the proper mapping in a subsequent PR.